### PR TITLE
Aea 278 move schema and examples to common directory

### PR DESCRIPTION
**Pull request by booshibooshi**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/policies/AssignMessage.AddCors.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Set>
         <Headers>
             <Header name="Access-Control-Allow-Origin">{request.header.origin}</Header>
-            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, nhsd-session-urid</Header>
+            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, if-match, nhsd-session-urid</Header>
```
Comment (conner-jowitt) [inline proxies/live/apiproxy/policies/AssignMessage.AddCors.xml:7]: Why have we added "if-match"?

File: .gitignore [hunk 2]
```diff
@@ -1,13 +1,14 @@
-node_modules/
-bin/
```
Comment (conner-jowitt) [inline .gitignore:2]: Are we no longer ignoring the bin folder?

File: README.md [hunk 3]
```diff
@@ -43,12 +44,17 @@ Various scripts and commands rely on environment variables being set. These are
...
- * `lint` -- Lints the spec and code
- * `build-spec` -- Outputs the specification as a **single file** into the `build/` directory
- * `run-spec-viewer` -- Serves a preview of the specification in human-readable format
- * `generate-examples` -- generate example objects from the specification
- * `validate` -- validate generated examples against FHIR R4
- * `run-coordinator` -- build and run the coordinator locally
+ * `test` -- Performs quality checks including linting, licence checking of dependencies and unit/low level integration tests
+ * `build` -- Outputs the FHIR R4 validated models and artifacts for the: specification, sandbox, coordinator and apigee proxies into the corresponding `dist/` directories
+ * `release` -- Pulls all the artifacts for the individual components together and arranges them in a format ready to deploy; used mainly by CI but useful to check the output matches expectations
+ * `clean` -- Removes the output from the build
```
Comment (conner-jowitt) [inline README.md:50]: Is it worth mentioning that it removes the output from "release" as well?

File: README.md [hunk 4]
```diff
@@ -64,13 +70,20 @@ npm t
 #### End-to-end tests
 To run e2e tests for the sandbox, you need to supply an environment. A `local` environment and an environment template are included under `tests/e2e/environments`.
 
-In order for local tests to work, you must have the sandbox server running locally.
+In order for tests under the make target `test-sandbox` to work, you must have the sandbox server running locally. In a seperate shell run:
 
 ```
 make run-sandbox
```
Comment (conner-jowitt) [inline README.md:76]: As the "run" command has a dependency on the "build" command, is it worth mentioning here to run it first? Likewise for the coordinator

File: azure-pipelines.yml [hunk 5]
```diff
@@ -82,36 +81,33 @@ jobs:
           restoreKeys: |
             "$(Agent.OS)" | "node-sandbox" | sandbox/package-lock.json
             "$(Agent.OS)" | "node-sandbox"
-
+
```
Comment (conner-jowitt) [inline azure-pipelines.yml:84]: Whitespace

File: scripts/build_proxies.sh [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+mkdir -p dist/proxies/sandbox
+mkdir -p dist/proxies/live
+cp -Rv proxies/sandbox/apiproxy dist/proxies/sandbox
+cp -Rv proxies/live/apiproxy dist/proxies/live
+mkdir -p dist/proxies/sandbox/apiproxy/resources/hosted
+mkdir -p dist/proxies/live/apiproxy/resources/hosted
+rsync -av --copy-links --exclude="node_modules" --filter=':- .gitignore' sandbox/ dist/proxies/sandbox/apiproxy/resources/hosted
+rsync -av --copy-links --filter=':- .gitignore' coordinator/dist/ dist/proxies/live/apiproxy/resources/hosted
+cp coordinator/package.json dist/proxies/live/apiproxy/resources/hosted
+cp coordinator/app.yaml dist/proxies/live/apiproxy/resources/hosted
```
Comment (conner-jowitt) [inline scripts/build_proxies.sh:18]: The other build commands are in the makefile, this is the only one in a seperate script. Should we move the others into scripts? Or should this be made part of the makefile?

File: .gitignore (hunk 2 above)
Comment (booshibooshi) [reply .gitignore:2]: yes, this was ignored due to the fhir-validation binary being downloaded here, this is now donwloaded under models/dist and ignored uner the **/dist/ rule

File: scripts/build_proxies.sh (hunk 6 above)
Comment (booshibooshi) [reply scripts/build_proxies.sh:18]: Think that's worth reviewing once we've used the new format for models for a while and see if we're happy with those changes. The scripts folder in general might be one to consider refactoring to help see where they are used. This script has been here a while, I just renamed it from build_proxy to build_proxies

File: proxies/live/apiproxy/policies/AssignMessage.AddCors.xml (hunk 1 above)
Comment (booshibooshi) [reply proxies/live/apiproxy/policies/AssignMessage.AddCors.xml:7]: Suspect it would be the result of a couple of merges, will remove

**Pull request merged**